### ISSUE TITLE
NVDA Remote fails to generate key

### ISSUE THREAD

**Issue by rkingett** [NVDARemote/NVDARemote#1]
I keep trying to generate a client key but the edit field is blank after I hit generate key. I also do not see it pop up in a small window if that is what I am missing

**Comment by ctoth** [comment]
Version 1.01, released a while ago, fixed this issue. You can download 
it from the add-on website.

On 7/3/2015 5:36 PM, rkingett wrote:
>
> I keep trying to generate a client key but the edit field is blank 
> after I hit generate key. I also do not see it pop up in a small 
> window if that is what I am missing
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NVDARemote/NVDARemote/issues/1>.
>


-- 
		-Q

	Accessible Software for the Blind
http://GetAccessibleApps.com

**Comment by leonardder** [comment]
It seems key generation still fails sometimes:
```ERROR - stderr (16:22:33):
Exception in thread Thread-6:
Traceback (most recent call last):
  File "threading.pyc", line 810, in __bootstrap_inner
  File "threading.pyc", line 763, in run
  File "C:\Users\Leonard\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\transport.py", line 42, in run
  File "ssl.pyc", line 844, in connect
  File "ssl.pyc", line 831, in _real_connect
  File "socket.pyc", line 228, in meth
error: [Errno 10061] No connection could be made because the target machine actively refused it```
